### ISSUE TITLE
Doesn't support creating `test`s with `forEach`  loop

### ISSUE THREAD

**Issue by selfrefactor** [vitest-dev/vscode#163]
It simply skips the tests when I request a run, even though tests are shown.
I think this is valid use case, as it is normal to want to apply DRY in our unit tests:

```
const TEST_DATA = [
  {label: 'episode', key: 'TEST-1'},
  {label: 'season', key: 'TEST-2'},
]

TEST_DATA.forEach(({label, key}) => {
  test(`with ${label}`, () => {
    let {item, level} = foo(getTestTreeData(), key)
    expect(item[ID]).toBe(key)
  })
})
```

**Comment by tjx666** [comment]
same here: https://github.com/tjx666/leetcode-javascript-2023/tree/main

**Comment by southorange0929** [comment]
same issue

**Comment by asnow-c** [comment]
`forEach` is similar to `test.each`
Similar issue: #7

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.
